### ISSUE TITLE
DisplayName is missing for a method without parameters

### ISSUE THREAD

**Issue by alisevych** [UnitTestBot/UTBotJava#664]
**Description**

When JUnit5 tests are generated for a method without input parameters - DisplayName annotation is missing.

**To Reproduce**

1. Run a project with JDK 8/11 in IDEA 2022.1.4
2. Add the following class:

```java
public class B {

    public String b = "B", c = "C";

    public String getBC() {
        return b + c ;
    }
}
```

3. Generate JUnit5 tests for it with No parametrization.
4. Open the generated test.

**Expected behavior**

Generated tests are supposed to contain DisplayName annotation with short test description.

**Actual behavior**

Only Test annotation is added to the generated method.

**Visual proofs (screenshots, logs, images)**

~~~java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class BTest {
    ///region Test suites for executable abc.ghi.B.getBC

    ///region

    @Test
    public void testGetBC() {
        B b = new B();

        String actual = b.getBC();

        String expected = "BC";
        assertEquals(expected, actual);
    }
    ///endregion

    ///endregion

}
~~~

**Environment**

Windows 10 Pro
IntelliJ IDEA 2022.1.4 Ultimate
Gradle project, JDK 11

**Additional context**

For a method with input parameter/s DisplayName annotation is added.

**Comment by alisevych** [comment]
Reproducing on the plugin build from main on August 01

**Comment by amandelpie** [comment]
Looks like the summaries are totally not generated including methodNames, JavaDocs and so on

**Comment by amandelpie** [comment]
@alisevych please mention or assign on me firstly the issues related to the summaries generation

**Comment by onewhl** [comment]
Hi @alisevych , it seems it couldn't be reproduced on fresh build.
I just updated my repo (last commit [25707e0624d99c8126e7ba18babc379864b71c26](https://github.com/UnitTestBot/UTBotJava/commit/25707e0624d99c8126e7ba18babc379864b71c26)), and ran UTBot on the code you provided. I got the following results:

~~~
    ///region Test suites for executable B.getBC

    ///region

    @Test
    @DisplayName("getBC: arg_0 = B()")
    public void testGetBC() {
        B b = new B();

        String actual = b.getBC();

        String expected = "BC";
        assertEquals(expected, actual);
    }
    ///endregion

    ///endregion
~~~
IntelliJ IDEA 2022.1.4 (Community Edition)
Build #IC-221.6008.13, built on July 19, 2022
Runtime version: 11.0.15+10-b2043.56 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
